portable/GCC/NiosII/port_asm.S

```diff
@@ -33,11 +33,10 @@
 .globl restore_sp_from_pxCurrentTCB		
 	
 # Entry point for exceptions.
-.section .exceptions.entry, "xa"		
+.section .exceptions.entry.user, "xa"

```

Comment:
I think that it may be possible to mess up if user change `.exceptions.entry.user` section. `.exceptions.entry.user` was used to certainly ignore default `.exceptions.entry` that BSP generated.  It is not needed to use `.exceptions.entry.user` if could certainly ignore default process.